spec/spec.md

```diff
@@ -682,12 +682,19 @@ Examples are listed inline in above sections as well as in complete form within
 
 - Microsoft
 - Workday
-- Mattr
 - Ping Identity
+- Gen Digital
+- Spruce Systems Inc.
+- Auth0Lab
 
 ## Testing
 
-Implementations may test conformance of the wallets to this profile using this [verification website](https://vcinteroptesting.azurewebsites.net/verifier).
+Implementations may test conformance of the wallets to this profile using the following verification websites:
+
+- https://vcinteroptesting.azurewebsites.net/verifier
+- https://verifiablecredentials.dev/presentation
+
```

Comment:
```suggestion
https://app.vp.interop.spruceid.xyz/

```

Reply:
could use a dash so it's part of the list